pyproject.toml

```diff
@@ -18,19 +18,25 @@ classifiers = [
 ]
 dependencies = [
     "PyCryptodome",
-    "aiohttp",  # TODO: remove in favor of httpx
+    "aiohttp",
+    # TODO: remove in favor of httpx
     "anyio",
-    "fhaviary[llm]>=0.6",  # For info on Message
-    "html2text",  # TODO: evaluate moving to an opt-in dependency
+    "fhaviary[llm]>=0.6",
+    # For info on Message
+    "html2text",
+    # TODO: evaluate moving to an opt-in dependency
     "httpx",
-    "litellm>=1.44",  # to prevent sys.stdout on router creation
+    "litellm>=1.44",
+    # to prevent sys.stdout on router creation
     "numpy",
     "pybtex",
     "pydantic-settings",
     "pydantic~=2.0",
     "pymupdf",
+    "python-docx>=1.1.2",
```

Comment:
We are trying to keep dependencies minimal, can you move this to an extra (see `project.optional-dependencies` below), and do lazy imports within `readers.py`